lib/YaST/workarounds.pm

```diff
@@ -0,0 +1,27 @@
+# Copyright 2015-2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+package YaST::workarounds;
+
+
+use strict;
+use warnings;
+use Exporter;
+use Mojo::Base 'Exporter', -signatures;
+use testapi;
+use version_utils;
+
+
+our @EXPORT = qw(workaround_screen_refresh);
+
+sub workaround_screen_refresh {
+    my ($needle) = shift;
+    if (!check_screen($needle, 180) && (is_sle('>=15-SP4'))) {
```

Comment:
condition should go outside, otherwise you cannot give a right name to the method and less you call it `check_and_apply_blablab`, it is more clear for read to see when something is applied from the call.
besides you should use the complex check on the right of the && operation to save time.

Reply:
normally workaround have in the name of the method the one that they apply, otherwise you need to dive into the code to figure out: `apply_workaround_bsc1204176`. There might be different refreshment issues so doesn't worth to include it in the name.

Reply:
check screen with 180? even in the documentation states that that is not recommended.
could you please comment the algorithm of the workaround, reading the code I don't get why we do it, perhaps in the bug or trying manually would make sense, but it is not clear from the code.

Reply:
The moment I did that lib I was replacing an assert screen that waited for 180. But maybe a better approach is to submit a timeout parameter that defaults to something much less and in the cases where it will take longer I can set up the timeout explicitly.